package.json

```diff
@@ -22,7 +22,7 @@
     "/ts3.4/"
   ],
   "scripts": {
-    "build": "tsc && downlevel-dts . ts3.4 && cp tsconfig.json ./ts3.4/",
+    "build": "tsc && rm -rf ./ts3.4 && downlevel-dts . ts3.4 && cp tsconfig.json ./ts3.4/",
```

Comment:
oof.

+1

Reply:
Yeah. I filed https://github.com/sandersn/downlevel-dts/issues/39 for this